client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -47,21 +37,20 @@ import withTrackingTool from 'lib/analytics/with-tracking-tool';
 import './style.scss';
 import { getTaskList } from 'my-sites/checklist/wpcom-checklist/wpcom-task-list';
 
-interface ChecklistTaskState {
-	[taskId: string]: {
-		completed: null | boolean;
-	};
-}
-
 interface Props {
 	isPremium: boolean;
 	isProfessional: boolean;
 	isPaidPlan: boolean;
-	taskStatuses: ChecklistTaskState | undefined;
+	taskStatuses:
+		| {
+				id: string;
+				isCompleted: boolean;
+		  }[]
+		| undefined;
 	widgetCustomizerPaneUrl: URL | null;
 }
 
-class JetpackChecklist extends PureComponent< Props > {
+class JetpackChecklist extends PureComponent< Props & LocalizeProps > {
```

Comment:
Just for my own educational purposes here, `Props & LocalizeProps` will just combine the 2 interfaces right? And allow for a shape that's somewhat like:

```js
{
  ...someProps,
  ...someOtherProps,
}
```

Right?

Reply:
Yep, I think that's a valid way of expressing it 👍 

This is an [_intersection type_](https://www.typescriptlang.org/docs/handbook/advanced-types.html#intersection-types).

Please feel free to question or add feedback about anything TypeScript 🙌